src/autocomplete.cpp

```diff
@@ -503,7 +517,7 @@ bool LocalFS::addCompletions(ACState& s)
         fs::path searchPath(s.word().s + (s.word().s.empty() || (s.word().s.back() == '\\'  || s.word().s.back() == '/' ) ? "*" : ""));
         char sep = (!s.word().s.empty() && s.word().s.find('/') != string::npos ) ?'/':'\\';
         bool relative = !searchPath.is_absolute();
-        searchPath = relative ? fs::current_path().append("\\").append(searchPath) : searchPath;
+        searchPath = relative ? fs::current_path().append("\\").append(searchPath.u8string()) : searchPath;
         std::string cp = relative ? fs::current_path().u8string() + "\\" : "";
```

Comment:
I think I only added it where one or another compiler insisted on it, perhaps the older experimental version of fs::path (which may differ a little between windows/linux libraries) is not completely consistent with the append() function.   Thanks very much for the review!

Reply:
I looked a bit further, and the path append() function really just converts the argument(s) to a path, and uses operator /= to do the actual joining.   So using that directly is shorter and more efficient, since we already have a path object to append.   I've made the change, and also enabled its use in megacli so that we can test it on linux (megaclc doesn't do anything with paths) (and provided the c++ compiler supports c++11, which mega::autocomplete needs).   cheers!